docs/website/docs/getting-started/pytorch.md

```diff
@@ -19,15 +19,13 @@ Install [`torch-mlir`](https://github.com/llvm/torch-mlir), necessary for
 compiling PyTorch models to a format IREE is able to execute:
 
 ```shell
-pip install -f https://llvm.github.io/torch-mlir/package-index/ torch-mlir
+pip install --pre torch-mlir \
+  -f https://llvm.github.io/torch-mlir/package-index/
+  --extra-index-url https://download.pytorch.org/whl/nightly/cpu
 ```
 
-A special `iree_torch` package makes it easy to compile PyTorch programs and
-run them on IREE:
-
-```shell
-pip install git+https://github.com/iree-org/iree-torch.git
-```
```

Comment:
Still a few links to the `iree-org/iree-torch` repo on this page, but these changes are a good improvement and I don't want to block them on a full repo restructuring. I'd like to pull some PyTorch (and JAX) samples into https://github.com/openxla/iree/tree/main/samples/colab at some point, since these APIs and packages have stabilized enough that some version drift is okay now.

Reply:
Yeah, my thinking was that it was better for now to have the links to some examples, even if they use `iree-torch`, rather than no links at all. Moving some of the examples into the `iree` repo in the future sounds like a good plan to me.